pkg/pillar/persistcache/README.md

```diff
@@ -0,0 +1,31 @@
+# Persisted Cache (persistcache)
+
+## Use-case
+
+This package implements a persistance cache. The main use-case is to store data on filesystem and
```

Comment:
Would be good to mention that the cache has key-value semantics.

Reply:
Fair :)

Reply:
Updated